### PR TITLE
feat(sdk-coin-zketh): add zketh sdk skeleton

### PR DESCRIPTION
**Pull request by ajays97**
Ticket: WIN-1417

This PR adds the new sdk-coin-zketh module.

### CODE REVIEW

**Review by zahin-mohammad — COMMENTED** [review]
Flush while in draft

**Review by ayush-9 — APPROVED** [review]

File: modules/sdk-coin-zketh/src/zketh.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  protected getTransactionBuilder(): EthLikeTransactionBuilder {
+    return new TransactionBuilder(coins.get(this.getBaseChain()));
+  }
+
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return false;
+  }
```
Comment (mohammadalfaiyazbitgo) [inline modules/sdk-coin-zketh/src/zketh.ts:25]: just double checking, why doesn't this support tss? I thought all new eth likes were tss enabled.

Comment (ajays97) [reply modules/sdk-coin-zketh/src/zketh.ts:25]: As per the current scope of the project, we do not support TSS wallets for zkSync. Also, there is nothing planned for the near future.

**Review by zahin-mohammad — APPROVED** [review]
Please also update the codeowners file for the new module (can be in a new PR)

File: modules/sdk-coin-zketh/src/zketh.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  /** @inheritDoc */
+  getMPCAlgorithm(): MPCAlgorithm {
+    return 'ecdsa';
+  }
+}
```
Comment (gianchandania) [inline modules/sdk-coin-zketh/src/zketh.ts:31]: There is no need for supportsTss and getMPCAlgorithm as we don't have tss support as of now. Please remove them in the follow up Pr that you are working on

**Review by gianchandania — APPROVED** [review]

File: modules/sdk-coin-zketh/package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "fmt": "prettier --write .",
+    "check-fmt": "prettier --check .",
+    "clean": "rm -r ./dist",
+    "lint": "eslint --quiet .",
+    "prepare": "npm run build",
+    "test": "npm run coverage",
+    "coverage": "nyc -- npm run unit-test",
+    "unit-test": "mocha"
+  },
+  "author": "Ajay Srinivas <ajsrinivas@bitgo.com>",
```
[the remark this answers is not in the document]
Comment (ajays97) [reply modules/sdk-coin-zketh/package.json:18]: Updated in the next followup PR.

File: modules/sdk-coin-zketh/src/zketh.ts (hunk 2 above)
Comment (ajays97) [reply modules/sdk-coin-zketh/src/zketh.ts:31]: Removed in the next PR.

**Pull request merged**